### ISSUE TITLE
Spurious touches reported

### ISSUE THREAD

**Issue by stonehippo** [BelaPlatform/Trill#7]
I've come across an issue with a couple my Trill boards (specifically a Bar and a Square, both acquired during the Kickstarter campaign). The sensors will report a touch that is not there after it has been running for a while. For example, on my Square, I am getting a horizontal touch at centroid location 1280. This seems to be consistent, and happens with both the Trill-Arduino firmware and a driver that I've written for CircuitPython.

I haven't looked at the firmware yet, but I'm going to take a read through to see how centroids are calculated. However, because this issue seems to appear on the sensor between different software implementations on different microcontrollers, I have to this it's the sensor itself having issues.

Is this something others have seen?

**Comment by giuliomoro** [comment]
Things that can cause ghost touches are moisture (e.g.: a thin layer of water on the surface or edge of the sensor) or something changing with the surroundings of the sensor. Is it resting on a flat surface, taped or glued down, or is it in a precarious position balancing on the cable sticking out of the back? Also remember that a sensor calibration is done automatically when the sensor is initialised by the sofware, so for instance if you have one finger on the sensor while the program starts and then you remove it you may have unexpected readings: you are not supposed to touch the sensor when the program starts.

**Comment by stonehippo** [comment]
It's definitely not a touch during calls to updateBaseline. I've got the Square mounted in the 3D printed base, but it was happening beforehand, so I suspect it might be something on the sensor. In the case of the Bar, I think it was maybe a spec of dust on the pins of the chip; removing it may have resolved the issue on that sensor. 

I do note that resetting the chip fixes the issue for a while (presumably because of the baseline update) but then it comes back, especially after a multiple touch. I'm going to dump the baseline for a while and see if I can map what's happening.

I've ordered a few more Trills for a project, and my CircuitPython library is almost complete, so I'll be spending plenty of time with them.
